### DISCUSSION TITLE
Problems with AutoCompleteBox not displaying dropdown.

### DISCUSSION THREAD

**Discussion by PrzemyslawPalinski** [AvaloniaUI/Avalonia#14373] [Q&A]
Like i said in title I'm having problems with getting AutoCompleteBox to display drop-down. It only happens when i set value of Text property in constructor of ViewModel connected to window. 

This is my axml element:
`<AutoCompleteBox Height="32" Background="#2e3440" Text="{Binding RequesterName, Mode=TwoWay}" ItemsSource="{Binding UserNames, Mode=OneWay}" IsEnabled="{Binding CanEditGrade}" FilterMode="Contains"/>`

and in ViewModel only thing i do is set value of RequesterName with value that comes from last window. When RequsterName is set to null problem does not occur. What am i doing wrong here?

**Comment by robloo** [comment]
I can confirm there are intermittent issues with `AutoCompleteBox` not showing the drop-down. I see the same issues with the drop down not opening in some cases while it works in others. Lately it isn't working at all though.

There is likely an issue with the control itself but I haven't investigated further. There were a few changes recently and I suspect we have more issues with them.

**Comment by timunie** [comment]
A minimal sample would be great to check what's the root cause here.

**Reply by robloo** [reply]
It's intermittent on my end and I didn't see it in ControlCatalog. It's only come up in an app and it's low on the priority list at the moment. Hopefully others can isolate this more.
